src/styles/generics/_fonts.scss

```diff
@@ -9,13 +9,17 @@
 }
 
 @font-face {
-  font-family: "Oxygen Bold";
+  font-family: Oxygen;
   src: url("../../../fonts/Oxygen/Oxygen-Bold.ttf");
+  font-weight: 700;
```

Comment:
Ah, having them both be the same name but different font-weights makes so much more since.

Reply:
Yeah, I think it'll make more sense in the long run! I should probably mention this to everyone else just so it doesn't cause too much confusion, though.